layouts/partials/menu.html

```diff
@@ -3,11 +3,7 @@
     <div id="header">
       {{ partial "logo.html" . }}
       {{ if .Site.Params.search }}
-      <div class="searchbox">
-        <label for="search-by"><i class="fa fa-search"></i></label>
-        <input id="search-by" type="text" placeholder="Search Documentation" data-search-input="/lunr.json/query">
-        <span data-search-clear=""><i class="fa fa-close"></i></span>
-      </div>
+        {{ partial "search.html" . }}
```

Comment:
It would be more flexible if  {{ partial "search.html" . }} is not in header class, like : 

```html
<nav id="sidebar">
  <div id="header-wrapper">
    <div id="header">
      {{ partial "logo.html" . }}
    </div>
    {{ if .Site.Params.search }}
      {{ partial "search.html" . }}
    {{ end }}
</div>
```